### PR TITLE
Add proposal batch update

### PR DESCRIPTION
**Pull request by pgobin**
- Add batch endpoint
- Add batch UI flow
- Address proposal selection issue

Addresses #788 and #37.

Signed-off-by: PGobz <p.gobin@gmail.com>

### CODE REVIEW

**Comment by PatrickLammers** [comment]
This should close out the Batch epic

File: rbac/server/api/proposals.py [hunk 1]
```diff
@@ -145,6 +145,16 @@ class ProposalType(object):  # pylint: disable=too-few-public-methods
     )
 
 
+@PROPOSALS_BP.patch("api/proposals")
+@authorized()
+async def batch_update_proposals(request):
```
Comment (nadiabahrami) [inline rbac/server/api/proposals.py]: Can you add a doc string to this so we know what it's for?

Comment (pgobin) [reply rbac/server/api/proposals.py]: Fixed. 😺

**Comment by pgobin** [comment]
> I can create a bug ticket regarding the Browse page not refreshing after a request has been approved.

That may be a separate issue. Could you create a bug for that with repro steps?

**Comment by mtn217** [comment]
@pgobin - Bug #1078 was created for the bug I found. Just need the version file updated and we can merge this in.

**Pull request merged**